src/pvrclient-nextpvr.cpp

```diff
@@ -660,8 +660,9 @@ bool cPVRClientNextPVR::CanSeekStream(void)
 {
   if (IsServerStreaming())
   {
+    //  function is not called for recordings
```

Comment:
The comment conflict with the code here. Maybe expand so it is clear.

Reply:
The documentation is not clear on how recordings deal with this.  Perhaps the best solution for now is to remove recording logic from that function entirely and assume it applies only to live tv to be consistent with other addons.

Reply:
Can seek stream is not for recordings at all. At least I never thought it was.

Reply:
Semantics I guess.  Here https://codedocs.xyz/AlwinEsch/kodi/group__cpp__kodi__addon__pvr___streams.html identifies 3 stream types.  If the function only applies to TV is probably should be under TV.

Reply:
Agreed the docs are not clear at all on that. Could you PR an update so we clean that up for matrix?

Reply:
NP I will have a look.